TEST_ENFORCEMENT.sh

```
@@ -0,0 +1,77 @@
+mkdir -p /tmp/test-packages/logger
+echo '{}' > /tmp/test-packages/logger/package.json
+cd /tmp/test-packages
+git init -q
+git add .
+if node /workspace/tools/validators/no-package-json-validator.js 2>&1 | grep -q "PACKAGE.JSON FILES FOUND"; then
+  echo "✅ PASS: package.json in packages/ detected"
+else
+  echo "❌ FAIL: package.json in packages/ not detected"
+fi
+rm -rf /tmp/test-packages
+cd /workspace
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)